cloud/gcp/main.tf

```diff
@@ -0,0 +1,38 @@
+provider "google" {
+  alias = "impersonation"
+  scopes = [
+    "https://www.googleapis.com/auth/cloud-platform",
+    "https://www.googleapis.com/auth/userinfo.email",
+  ]
+}
+
+data "google_service_account_access_token" "default" {
+  provider               = google.impersonation
+  target_service_account = var.terraform_service_account
+  scopes                 = ["userinfo-email", "cloud-platform"]
```

Comment:
do we need this scope?

Reply:
Yes. Terraform service account wasn't able to assume identity without it